apps/emqx_conf/src/emqx_conf_schema.erl

```diff
@@ -684,18 +684,18 @@ fields("cluster_call") ->
             )},
         {"max_history",
             sc(
-                range(1, 500),
+                range(100, 10240),
```

Comment:
nit: There may be compatibility issues here? If the previous configuration used values from 1 to 99.